src/ansys/geometry/core/connection/launcher.py

```diff
@@ -236,6 +237,15 @@ def launch_remote_modeler(
 
     Parameters
     ----------
+    platform : str, default: None
+        **Specific for Ansys Lab**. The platform option for the Geometry service.
+        The default is ``"windows"``.
```

Comment:
Is the default really `windows`? It seems to be None based on line 224

Reply:
It's ``None`` by argument. But by behavior it should be "windows". Let me explicitly add it - I agree.

Reply:
OK, got it. My bad.  It makes sense.

Reply:
No worries - it was a good point nonetheless. I added the default string on argument. Thanks for catching it!